docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,304 @@
+# Validation Criteria Master Document
+This document contains a comprehensive list of validation criteria referenced by all Test Instructions.
+
+Its objective is to guarantee consistency of criteria, allowing the authors to refer to the same table format.
+
+> **ATTENTION**: for some rules a test-specific VALUE might be needed. In this case the tables below just capture an example between []
+
+
+
+## Utilities
```

Comment:
- I am a bit confused with the heading "Utilities". This section seems to correspond with what we (I) did in the Readme on Dataset level i.e. a dataset specification.

Reply:
Let's find another name. It is meant to host the templates for the tables of all the rules that say "As per XXX Table". @apinzenoehler requested it for an easier copy-paste.

Reply:
Changed it to "Table Templates" until further decision